soepy/test/test_unit.py

```diff
@@ -212,4 +215,18 @@ def test5():
     np.testing.assert_array_equal(states_true, states[0:48, :])
 
 
+def test6():
+    """This test ensures that the examples that are parts of the docstrings work
+    as intended.
+    """
+    np.testing.assert_equal(
+        subprocess.call(
+            ["python", "-m", "doctest", "solve_auxiliary.py"],
```

Comment:
Please consider the following:

- Can we please set this up so that all doctests anywhere in the package are tested and we do not need to add files here `solve_auxiliary.py`. 
-  Is it possible to call the doctest module directly as we do here for pytest https://github.com/OpenSourceEconomics/soepy/blob/e2708186c463bd5a17e43fead2c8132018b69b88/soepy/__init__.py#L15 The current setup is not working on Windows for example.

Reply:
I have tackled this and integrated doctest as an additional argument in pytest.ini.